tempus_fugit_minecraft/main.py

```diff
@@ -448,6 +442,14 @@ def generate_clouds(n, num_of_clouds=250):
                     single_cloud.append((x,cloud_center_y,z))
             clouds.append(single_cloud)
         return clouds
+    
+    
+    def draw_clouds_in_the_sky(self, clouds):
```

Comment:
I think a better name for this would be something like `place_cloud_blocks`

Reply:
> I think a better name for this would be something like `place_cloud_blocks`

Agreed